controllers/datasource_controller.go

```diff
@@ -136,19 +137,18 @@ func (r *GrafanaDatasourceReconciler) syncDatasources(ctx context.Context) (ctrl
 		// so we should minimize those updates
```

Comment:
I don't think this comment is accurate anymore

Reply:
The `WithEventFilter(ignoreStatusUpdates()).` line has not been added to the Grafana controller as of yet.
That is still TBD!